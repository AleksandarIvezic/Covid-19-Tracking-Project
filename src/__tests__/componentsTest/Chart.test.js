import React from 'react';
import renderer from 'react-test-renderer';
import LineChart from '../../components/Chart';

describe('LineChart', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <LineChart
          chartData={[545, 345]}
          country="serbia"
          type="totalRecovered"
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
