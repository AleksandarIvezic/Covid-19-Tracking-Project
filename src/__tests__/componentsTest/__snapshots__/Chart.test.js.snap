// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineChart renders correctly 1`] = `
Array [
  <div
    className="d-flex stripe-text justify-content-between"
  >
    <span
      className="m-1"
    >
      LAST 7 DAYS STATS
    </span>
  </div>,
  <canvas
    data-testid="canvas"
    height={150}
    onClick={[Function]}
    role="img"
    width={300}
  />,
]
`;
